picard/ui/edittagdialog.py

```diff
@@ -22,24 +23,55 @@
     QtWidgets,
 )
 
+from picard.const import (
+    RELEASE_COUNTRIES,
+    RELEASE_FORMATS,
+    RELEASE_PRIMARY_GROUPS,
+    RELEASE_SECONDARY_GROUPS,
+    RELEASE_STATUS,
+)
 from picard.util.tags import TAG_NAMES
 
 from picard.ui import PicardDialog
 from picard.ui.ui_edittagdialog import Ui_EditTagDialog
 
 
+AUTOCOMPLETE_RELEASE_TYPES = [s.lower() for s
+                              in sorted(list(RELEASE_PRIMARY_GROUPS.keys())) + sorted(list(RELEASE_SECONDARY_GROUPS.keys()))]
```

Comment:
Since `RELEASE_*_GROUPS` are dictionnaries, `.keys()` shouldn't be needed, and `list()` neither:

```python
AUTOCOMPLETE_RELEASE_TYPES = [s.lower() for s in sorted(RELEASE_PRIMARY_GROUPS) + sorted(RELEASE_SECONDARY_GROUPS)]
```

Reply:
Thanks, fixed this